go/vumitools/account/models.py

```diff
@@ -42,11 +42,18 @@ class UserAccount(Model):
     event_handler_config = Json(default=list)
     msisdn = Unicode(max_length=255, null=True)
     confirm_start_conversation = Boolean(default=False)
-    disable_optouts = Boolean(default=False)
-    can_manage_optouts = Boolean(default=False)
     email_summary = Unicode(max_length=255, null=True)
     tags = Json(default=[])
     routing_table = RoutingTableField(default=RoutingTable({}))
+    flags = ListOf(Unicode())
+
+    @property
+    def can_manage_optouts(self):
+        return u'can_manage_optouts' in self.flags
```

Comment:
Having these as properties does mean that anything that tries to set them manually in the current codebase won't work anymore. I don't that is the case, but we can check if tests fail.

This also means that setting flags in any new code needs to be done via the `flags` field. Should we add a method for each flag for doing the setting part? Not sure how necessary that is.

@jerith @hodgestar @rud thoughts?